.github/workflows/ci.yml

```diff
@@ -51,6 +51,18 @@ jobs:
           path: oas-examples-repo
           repository: readmeio/oas-examples
 
+      # For every GitHub Action release, we deploy our Docker images
+      # to the GitHub Container Registry for performance reasons.
+      # Instead of testing against the pre-built image, we can build
+      # an image from the currently checked-out repo contents by doing a
+      # li'l update in the action.yml file to point to the current Dockerfile.
+      - name: Replace Docker image value in action.yml
+        uses: jacobtomlinson/gha-find-replace@v3
+        with:
+          find: 'image:.*'
+          replace: "image: 'Dockerfile'"
+          include: rdme-repo/action.yml
```

Comment:
Why not call `bin/set-action-image.sh` instead?

Reply:
ah I hadn't thought of that — this is pretty similar. issue is we'd need to add a step to install deps since we use `js-yaml` and that doesn't feel totally worth it 😕